test/navigation.spec.js

```diff
@@ -28,6 +28,10 @@ module.exports.describe = function({testRunner, expect, playwright, FFOX, CHROMI
       await page.goto(server.EMPTY_PAGE);
       expect(page.url()).toBe(server.EMPTY_PAGE);
     });
+    fit('should use http for no protocol', async({page, server}) => {
```

Comment:
Shouldn't this test localhost url? Also, no fit please :)

Reply:
Yeah, bot's aren't happy with fits!